modules/041-linstor/images/linstor-server/liveness.sh

```diff
@@ -21,6 +21,20 @@ fi
 
 # Sometimes nodes can be shown as Online without established connection to them.
 # This is a workaround for https://github.com/LINBIT/linstor-server/issues/331, https://github.com/LINBIT/linstor-server/issues/219
+
+# Collect list of satellite nodes
+SATELLITES_ONLINE=$(linstor -m --output-version=v1 n l | jq -r '.[][] | select(.type == "SATELLITE" and .connection_status == "ONLINE").name' || true)
```

Comment:
```suggestion
SATELLITES_ONLINE=$(linstor -m --output-version=v1 node list | jq -r '.[][] | select(.type == "SATELLITE" and .connection_status == "ONLINE").name' || true)
```
And why "|| true"?

Reply:
It's for possible timeout while executing command, but I think it can be unnecessary.